### PR TITLE
Backlog 13743 migrated build and publish; added header to readme

### PR DESCRIPTION
**Pull request by astritademi62**
Migrated build and publish to circleci; added a badges header to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+|  | Badges | 
+| --- | --- |
+| Module | ![ID](https://img.shields.io/badge/ID-sdl--generator--tools-blue) [![Store](https://img.shields.io/badge/Jahia%20Store-Yes-brightgreen)](https://store.jahia.com/contents/modules-repository/org/jahia/modules/sdl-generator-tools.html) |
+| CI / CD | [![CircleCI](https://circleci.com/gh/Jahia/sdl-generator-tools/tree/master.svg?style=shield&circle-token=b8a70b01d5e9483ebbac3babfa26ae80930a56c7)](https://app.circleci.com/pipelines/github/Jahia/sdl-generator-tools) ![Unit Tests](https://img.shields.io/badge/Unit%20Tests-No-red) ![Integration Tests](https://img.shields.io/badge/Integration%20Tests-No-red) ![Build Snapshot](https://img.shields.io/badge/Build%20Snapshot-Yes-brightgreen) ![Build Release](https://img.shields.io/badge/Build%20Release-No-red) |
```
Comment (Fgerthoffert) [inline README.md]: You can remove the status token for circleci since this repo is public

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+|  | Badges | 
+| --- | --- |
+| Module | ![ID](https://img.shields.io/badge/ID-sdl--generator--tools-blue) [![Store](https://img.shields.io/badge/Jahia%20Store-Yes-brightgreen)](https://store.jahia.com/contents/modules-repository/org/jahia/modules/sdl-generator-tools.html) |
```
Comment (Fgerthoffert) [inline README.md:3]: Is this module published on the store ?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+| CI / CD | [![CircleCI](https://circleci.com/gh/Jahia/sdl-generator-tools/tree/master.svg?style=shield&circle-token=b8a70b01d5e9483ebbac3babfa26ae80930a56c7)](https://app.circleci.com/pipelines/github/Jahia/sdl-generator-tools) ![Unit Tests](https://img.shields.io/badge/Unit%20Tests-No-red) ![Integration Tests](https://img.shields.io/badge/Integration%20Tests-No-red) ![Build Snapshot](https://img.shields.io/badge/Build%20Snapshot-Yes-brightgreen) ![Build Release](https://img.shields.io/badge/Build%20Release-No-red) | 
+| Artifacts | [![Snapshot](https://img.shields.io/badge/Snapshot-Nexus-blue)](https://devtools.jahia.com/nexus/content/repositories/jahia-enterprise-snapshots/org/jahia/modules/sdl-generator-tools/) [![Release](https://img.shields.io/badge/Release-Nexus-blue)](https://devtools.jahia.com/nexus/content/repositories/jahia-enterprise-releases/org/jahia/modules/sdl-generator-tools/) |
```
Comment (Fgerthoffert) [inline README.md:5]: Artifact URL doesn't seem correct

File: README.md (hunk 2 above)
Comment (astritademi62) [reply README.md:3]: hmm, I guess not

File: README.md (hunk 1 above)
Comment (astritademi62) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (astritademi62) [reply README.md:5]: I just checked and its not there so I'm not sure where its being deployed to

**Pull request merged**